Add 2.22.0 release (#125)
percona/community: content/blog/2021-09-16-2-22-0-preview.md

---
title: "2.22.0 Preview Release"
date: "2021-09-16T00:00:00+00:00"
tags: ['PMM_prev']
authors:
  - denys_kondratenko
images:
  - superhero.png
slug: preview-release
---

## 2.20.0 Preview Release

Percona Monitoring and Management 2.22.0 is released today as a Community preview.

PMM team really appreciates your feedback!

Known issue:

> https://jira.percona.com/browse/PMM-8829

Reviewer: ```suggestion
- [PMM-8829](https://jira.percona.com/browse/PMM-8829) - "Missing Listen Port" error for external exporters after restart
```

Release notes:

> https://deploy-preview-588--pmm-doc.netlify.app/release-notes/2.22.0.html

Reviewer: ```suggestion
Release Notes Preview found [here](https://deploy-preview-588--pmm-doc.netlify.app/release-notes/2.22.0.html)
```

### PMM server docker

> https://www.percona.com/doc/percona-monitoring-and-management/2.x/setting-up/server/docker.html

Reviewer: ```suggestion
[Instructions](https://www.percona.com/doc/percona-monitoring-and-management/2.x/setting-up/server/docker.html)
```

docker tag: `perconalab/pmm-server:2.22.0-rc`

https://hub.docker.com/layers/perconalab/pmm-server/2.22.0-rc/

### PMM client package installation

Download the latest pmm2-client Release Candidate tarball for 2.22.0 from this [link](https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-2003-7917413.tar.gz).

If you want to install pmm2-client package, please enable testing repository via Percona-release: 
```
percona-release enable original testing
```

install pmm2-client package for your OS via package manager

### OVA 

> https://www.percona.com/doc/percona-monitoring-and-management/2.x/setting-up/server/virtual-appliance.html

Reviewer: ```suggestion
[Instructions](https://www.percona.com/doc/percona-monitoring-and-management/2.x/setting-up/server/virtual-appliance.html)
```

Artifact: [PMM2-Server-2021-09-14-1514.ova](http://percona-vm.s3-website-us-east-1.amazonaws.com/PMM2-Server-2021-09-14-1514.ova)

### AMI 

https://www.percona.com/doc/percona-monitoring-and-management/2.x/setting-up/server/aws.html

Artifact id: `ami-0a6b861c9225afbd8`

---

> Please also check out our Engineering Monthly Meetings https://percona.community/contribute/engineeringmeetings/ and join us on our journey in OpenSource! Contact us in [per.co.na/discord](https://per.co.na/discord).

Reviewer: Need to hint to people what to do when done testing?  
disable testing repo...roll back client version
maybe recommend test in a sandbox/vm vs on your live system?

Author: added a hint

---
title: "2.22.0 Preview Release"
date: "2021-09-16T00:00:00+00:00"
tags: ['PMM_prev']
authors:
  - denys_kondratenko
images:
  - superhero.png
slug: preview-release
---

## 2.20.0 Preview Release

> Percona Monitoring and Management 2.22.0 is released today as a Community preview.

Reviewer 2: What does 'Community preview' mean? How is it different to a normal release?

Author: no idea, this is first try. we will come up with something better.
it is like Early Preview probably.

Author: what is the better name ?

Reviewer 3: ```suggestion
Percona Monitoring and Management 2.22.0 is released today as a Preview Release.
```

PMM team really appreciates your feedback!

> Please try it out only in **testing environment**, those packages and images aren't supposed to be deployed in production systems. Final version is expected to be released through the standard channels in a coming week.

Reviewer 2: ```suggestion
We encourage you to try this PMM community release in **testing environments** only, as these packages and images are not fully production-ready. The final version is expected to be released through the standard channels in the coming week.
```

> We encourage you to try this PMM community release in **testing environments** only, as these packages and images are not fully production-ready. The final version is expected to be released through the standard channels in the coming week.

Reviewer 3: ```suggestion
We encourage you to try this PMM Preview Release in **testing environments** only, as these packages and images are not fully production-ready. The final version is expected to be released through the standard channels in the coming week.
```
